zipline/data/minute_bars.py

```diff
@@ -636,7 +636,7 @@ def convert_col(col):
         table.flush()
 
 
-class BcolzMinuteBarReader(object):
```

Comment:
Talked offline with @llllllllll. Going to roll back the part of this PR which creates separate reader classes, in favor of one class which uses parameters. The `WithBcolzEquityMinuteBarReader` and `WithBcolzFutureMinuteBarReader` fixtures will be responsible for setting the respective minutes per day values.